### DISCUSSION TITLE
Adding a package by user/repo

### DISCUSSION THREAD

**Discussion by jiriks74** [nix-community/kickstart-nix.nvim#20] [Q&A]
First of all thanks for making this! NixOS is hard enough to keep working and setting up NeVim using the other ways available is way harder than this. Thanks for keeping me running while I learn how to use Nix properly.

Now to the point. I need to use a fork of a plugin since upstream repo is dead. I added my plugins from nixpkgs until now so I didn't deal with this.

I know that in the readme you wrote:
>By adding plugins as flake inputs (if you like living on the bleeding-edge). Plugins added as flake inputs must be built in `nix/plugin-overlay.nix`.

But I don't see a demo file and I have no idea how to set that file up. This is the first flake  I came into contact with so I don't really understand them (Nix is really hard considering there's like 10 different places with the "official" docs now add flakes to that...].

Would you please show me a sample file or something so that I can start with that?

**Comment by mrcjkb** [comment]
Hey :wave: 

Thanks for the nice feedback :smile: 

[Here's a flake](https://github.com/pierrot-lc/nvim-nix) based on kickstart-nix.nvim that adds a bunch of plugins (and neovim nightly) as flake inputs.
